frontend/src/components/SessionPlayerTile.js

```diff
@@ -0,0 +1,37 @@
+import styled from 'styled-components/macro';
+import { Gi3DMeeple } from 'react-icons/all';
+import { IconContext } from 'react-icons';
+
+export default function SessionPlayerTile({ player }) {
```

Comment:
There is some duplication with `PlayerTile`, maybe extract the common behavior to a component?

Reply:
so I did that, but how do I access other elements now that I don't have a styled component part in the actual component? the text alignment seems to be off I little bit now...